includes/User.php

```diff
@@ -3173,34 +3143,29 @@ public function doLogout() {
 	 * @todo Only rarely do all these fields need to be set!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)